sdk/ml/azure-ai-ml/azure/ai/ml/operations/_data_operations.py

```diff
@@ -344,8 +344,15 @@ def _get_latest_version(self, name: str) -> Data:
         Latest is defined as the most recently created, not the most
         recently updated.
         """
-        result = _get_latest(name, self._operation, self._resource_group_name, self._workspace_name)
-        return Data._from_rest_object(result)
+        latest_version = _get_latest_version_from_container(name, self._container_operation, self._resource_group_name, self._workspace_name)
+        data_version_resource = self._operation.get(
```

Comment:
Minor: you can just call `self.get()`.

Reply:
Thought of doing it earlier but avoided because of recursion. Updated it.